### PR TITLE
docs(debug): add information about profiling

### PR DESCRIPTION
**Pull request by drodil**
## Hey, I just made a Pull Request!

Add information to documentation how to profile and stress test Backstage application.

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [x] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Comment by drodil** [comment]
@freben @Rugvip as discussed yesterday :)

File: docs/tooling/local-dev/profiling.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        options:
+          token: autocannon12345
+          subject: autocannon
+```
+
+See more information in the [Service to Service Auth](../../auth/service-to-service-auth.md) documentation.
+To run the stress test, you can use the following command:
+
+```shell
+autocannon -H "Authorization=Bearer autocannon12345" http://localhost:7007/api/catalog/entities
```
Comment (freben) [inline docs/tooling/local-dev/profiling.md:53]: With an equals sign for -H? Curious that they made it different from curl :)

**Review by freben — COMMENTED** [review]
Hi, this is fantastic! Remember to add it to sidebars.json too.

File: docs/tooling/local-dev/profiling.md (hunk 1 above)
Comment (drodil) [reply docs/tooling/local-dev/profiling.md:53]: It's a Header flag. Yeah it's a bit different from what people coming from curl world are used to :D

**Pull request merged**